lib/hypernova/controller_helpers.rb

```diff
@@ -88,7 +88,7 @@ def hypernova_batch_after
           result = @hypernova_batch.submit!
           on_success(result, hash)
         rescue StandardError => e
-          on_error(e)
+          on_error(e, {}, hash)
```

Comment:
Without a breaking change to `on_error`, I can't come up with a non-weird way to do this. perhaps we should make the second argument become a hash of "job" and "hash"?